cabal-helper.cabal

```diff
@@ -135,7 +135,6 @@ library c-h-internal
                        CabalHelper.Runtime.HelperMain
                        CabalHelper.Shared.Common
                        CabalHelper.Shared.InterfaceTypes
-  other-modules:
```

Comment:
Keep that so it doesn't turn into an exposed-module by accident when we go back to the internal lib.